server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -1535,7 +1535,8 @@ public TConditionalSession startConditionalUpdate(TInfo tinfo, TCredentials cred
         try {
           TabletServer.this.resourceManager.waitUntilCommitsAreEnabled();
         } catch (HoldTimeoutException hte) {
-          // Assumption is that the client has timed out and is gone. If thats not the case throw an
+          // Assumption is that the client has timed out and is gone. If that's not the case throw
+          // an
```

Comment:
Could combine this line with the next line.

Reply:
I don't recall changing that. Might have been the formatter. I'll move the an to the next line

Reply:
It was probably from a recent commit. Thanks for the fix!